src/request/connect/index.html

```diff
@@ -0,0 +1,133 @@
+    <link rel="stylesheet" bundle-toplevel href="../../nimiq-style.css">
+    <link rel="stylesheet" bundle-toplevel href="../../common.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PasswordInput.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PasswordBox.css">
+
+    <link rel="stylesheet" href="../../components/Tooltip.css">
+    <link rel="stylesheet" href="../../components/LoginFileAccountIcon.css">
+    <link rel="stylesheet" href="Connect.css">
+</head>
+<body class="loading">
+    <div class="header-top">
+        <div class="nimiq-app-name">
+            <span class="nimiq-logo"></span>
+        </div>
+    </div>
+
+    <div id="app">
+        <div id="connect-account" class="page nq-card">
+            <div class="page-header nq-card-header">
+                <h1 class="nq-h1">
+                    <span data-i18n="connect-heading-prefix">Log in to</span>
+                    <span class="app-name"></span>
+                </h1>
+                <span class="more">
+                    Connect and use your account to<br>
+                    approve transactions.
+                </span>
+            </div>
+
+            <div class="page-body nq-card-body">
+                <div class="app-icon-account-icon">
+                    <img class="app-icon">
+                    <div class="login-file-account-icon"></div>
+                </div>
+
+                <div class="flex-grow"></div>
+
+                <div class="bottom-explainer">
+                    The connection is for approving only.<br>
+                    Your funds remain unaffected.
+                    <div class="tooltip top" tabindex="0">
+                        <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-info-circle-small"/></svg>
+                        <div class="tooltip-box">
+                            <p data-i18n="connect-tooltip-multisig-1">
+                                Your account will be used to verify your identity only.
+                            </p>
+                            <p data-i18n="connect-tooltip-multisig-2">
+                                The shared wallet has its own address. It can receive transactions from anywhere just like a regular wallet.
+                            </p>
+                            <p data-i18n="connect-tooltip-multisig-3">
+                                The first address of your account will be visible to all participants in your shared wallet.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="page-footer nq-card-footer">
+                <button data-i18n="passwordbox-connect-account" class="nq-button light-blue continue display-none">Connect your account</button>
+                <form id="password-box" class=""></form>
```

Comment:
Empty `class` can be removed.

```suggestion
                <form id="password-box"></form>
```

Reply:
Resolved in #524